extensions-core/stats/src/main/java/io/druid/query/aggregation/teststats/ZtestPostAggregator.java

```diff
@@ -46,79 +48,92 @@
    http://facweb.cs.depaul.edu/sjost/csc423/documents/test-descriptions/indep-z.pdf
 */
 @JsonTypeName("zscore2sample")
-public class ZtestPostAggregator implements PostAggregator {
+public class ZtestPostAggregator implements PostAggregator
```

Comment:
The class name follows the naming convention of all Druid PostAggregators.
The Json Type Name just tried to be brief and meaningful.

Reply:
The class name tries to be brief while following the naming convention of all Druid postAggregators.
The Type Name just tries to be meaningful in the context of test statistics.

Reply:
I mean, why the class name can't be simply ```ZScore2SamplePostAggregator```? This is maybe a silly question because I'm not familiar with stats.

Reply:
There are various test stats can be calculated based on z-test.  It' not clear whether we should have one class for every variation of z-test based stats.  For example, it might make more sense to have 1-sample and 2-sample zscore in one class, rather than in two different classes.  In other words, I'm not thinking in terms of naming convention, but rather more along the line of extensibility and componentization.

Reply:
It sounds good to me, but if so, the json type name should also be an extendable name, right?

Reply:
The JsonTypeName (zscore2sample) is intended to be brief and meaningful to the user, so it is different from the class name ZtestPostAggregator.  Not sure what you mean by "the json type name should also be an extendable name"?  But if you mean we should make the type name in a way so that we can easily add a new type in a consistent manner; then yes.  JsonTypeName is designed from the user perspective and is exposed to the user, while the class name is to do with implementations.  The rationales behind these namings are different.